test/unit/mocks.ts

```diff
@@ -24,7 +24,7 @@
  **********/
 
 import RedisMock, { RedisClusterOptions, RedisOptions } from 'ioredis-mock';
-import { BasicConnectionConfig } from '#src/types';
+import { BasicConnectionConfig } from '../../src/types';
```

Comment:
is it necessary to do such import changes?

Reply:
@geka-evk Yes, it breaks code navigation for VS Code.